### PR TITLE
[AdminBundle] New login styling

### PR DESCRIPTION
**Pull request by dbeerten**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | yes
| BC breaks?    | no
| Deprecations? | no
| Fixed tickets |

### CODE REVIEW

**Review by ProfessorKuma — CHANGES_REQUESTED** [review]
Hi @, your PR needs some changes
* Your answer if this PR is a bugfix seems to be incorrect.
* Your answer if this PR is a new feature seems to be incorrect.
* Your answer if this PR is BC breaking seems to be incorrect.
* Your answer if this PR is has deprecations seems to be incorrect.
* your PR title should look like `[SomeBundle] Fixed some code`

File: src/Kunstmaan/AdminBundle/Resources/views/Default/layout.html.twig [hunk 1]
```diff
@@ -11,11 +11,28 @@
...
+
+    <!-- Customizable Brand Color -->
+    {# <style type="text/css">
+    :root {
+        --highlight-color-bg: #90268E;
+        --highlight-color-fg: #ffffff;
+    }
+</style> #}
+
+{#    {% include "@KunstmaanAdmin/Default/_css_extra.html.twig" %}#}
```
Comment (acrobat) [inline src/Kunstmaan/AdminBundle/Resources/views/Default/layout.html.twig:28]: Do we need to uncomment or remove this line? (and uncomment line 35?)

File: src/Kunstmaan/AdminBundle/Resources/views/authentication/email/password_reset.html.twig [hunk 2]
```diff
@@ -0,0 +1,196 @@
+{# TODO: html template, teksten in translations #}
```
Comment (acrobat) [inline src/Kunstmaan/AdminBundle/Resources/views/authentication/email/password_reset.html.twig:1]: Remove todo comment

File: src/Kunstmaan/AdminBundle/Resources/views/authentication/login.html.twig [hunk 3]
```diff
@@ -1,6 +1,9 @@
 {% extends '@KunstmaanAdmin/authentication/layout.html.twig' %}
 
 {% block login_content %}
+    <h2 class="app-login__form-title">
+        Log in to {{ websitetitle | trans }}
```
Comment (acrobat) [inline src/Kunstmaan/AdminBundle/Resources/views/authentication/login.html.twig:5]: We need to move this to a translation key

File: src/Kunstmaan/AdminBundle/Resources/views/authentication/login.html.twig [hunk 4]
```diff
@@ -14,43 +17,46 @@
...
-            {{ 'security.login.submit'|trans }}
-        </button>
+        <div class="form-group app-login__form__submit-holder">
+            <button type="submit" id="_submit" name="_submit" class="btn btn--md btn--inverted">
+                {{ 'security.login.submit'|trans }}
+            </button>
+        </div>
     </form>
 
     {% if google_signin_enabled() %}
```
Comment (acrobat) [inline src/Kunstmaan/AdminBundle/Resources/views/authentication/login.html.twig:57]: Can you check if the signin with google button also show correct in the new layout? (Can be temporary "faked" by commenting the if statement)

File: src/Kunstmaan/AdminBundle/Resources/views/authentication/layout.html.twig [hunk 5]
```diff
@@ -1,5 +1,68 @@
...
+                        </div>
+                    {% endfor %}
+                </div>
+                {% block login_content %}{% endblock %}
+            </div>
+
+            {% block login_footer %}
+                <footer class="app-login__footer">
+                    <p class="app-login__footer__text">
+                        <img src="/bundles/kunstmaanadmin/default-theme/img/kunstmaan/logo-cms-color.svg" class="app-login__footer__logo">
```
Comment (acrobat) [inline src/Kunstmaan/AdminBundle/Resources/views/authentication/layout.html.twig:55]: Can you add the `asset` function around the image path?

File: src/Kunstmaan/AdminBundle/Resources/views/authentication/layout.html.twig [hunk 6]
```diff
@@ -1,5 +1,68 @@
...
+                {% if login_logo is defined %}
+                    <img class="app-login__brand-logo" src="{{ asset(login_logo) }}" alt="{{ websitetitle | trans }}">
+                {% endif %}
+                <h1 class="app-login__title">
+                    {{ websitetitle | trans }}
+                </h1>
+            </header>
+
+            <figure class="app-login__image-holder">
+                {% set login_image_url = '/bundles/kunstmaanadmin/default-theme/img/kunstmaan/default-login-bg.jpg' %}
```
Comment (acrobat) [inline src/Kunstmaan/AdminBundle/Resources/views/authentication/layout.html.twig:18]: Can you add the `asset` function around the image path?

File: src/Kunstmaan/AdminBundle/Resources/views/authentication/layout.html.twig [hunk 7]
```diff
@@ -1,5 +1,68 @@
...
+                <img src="{{ login_image_url }}" alt="">
+            </figure>
+        </div>
+        <aside class="app-login__aside">
+            <div class="app-login__aside__content">
+                <div class="app-login__alert-holder">
+                    {% if is_granted("IS_AUTHENTICATED_REMEMBERED") %}
+                        <div class="alert alert-warning">
+                            <strong>{{ 'form.warning' | trans }}: </strong>
+                            {{ 'layout.logged_in_as'|trans({'%username%': app.user.username}, 'FOSUserBundle') }}
```
Comment (acrobat) [inline src/Kunstmaan/AdminBundle/Resources/views/authentication/layout.html.twig:31]:
```suggestion
                            {{ 'layout.logged_in_as'|trans({'%username%': app.user.username}) }}
```

File: src/Kunstmaan/AdminBundle/Resources/views/authentication/layout.html.twig [hunk 8]
```diff
@@ -1,5 +1,68 @@
...
+                            <a href="{{ path('kunstmaan_admin_logout') }}" class="btn btn-warning alert__action">
+                                {{ 'layout.logout'|trans({}, 'FOSUserBundle') }}
```
Comment (acrobat) [inline src/Kunstmaan/AdminBundle/Resources/views/authentication/layout.html.twig:33]:
```suggestion
                                {{ 'layout.logout'|trans }}
```

**Review by acrobat — CHANGES_REQUESTED** [review]

**Review by ProfessorKuma — CHANGES_REQUESTED** [review]
Hi @, your PR needs some changes
* This PR seems to need a milestone of a minor release.

**Review by ProfessorKuma — APPROVED** [review]
Hi @, your PR passed all our requirements.

Thank you for contributing!

**Review by acrobat — APPROVED** [review]

**Pull request merged**